### PR TITLE
Make placeholder optional + Remove editor on scope destroy

### PR DESCRIPTION
**Pull request by sebastiannm**
Hi,

I'm making here a pull request with stuff that were giving me problems on my side.

- Make the placeholder optional
If placeholder is set to false, it would be errors

- Remove editor if scope gets removed
Cleaning up after the scope gets destroyed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-medium-editor",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (simison) [inline package.json]: Perhaps this should be left out for @thijsw to do.

File: dist/angular-medium-editor.js [hunk 2]
```diff
@@ -46,6 +49,10 @@ angular.module('angular-medium-editor', [])
         scope.$watch('bindOptions', function(bindOptions) {
           ngModel.editor.init(iElement, bindOptions);
         });
+
+        scope.$on('$destroy', function(){
+          ngModel.editor.destroy();
+        });
```
Comment (simison) [inline dist/angular-medium-editor.js]: Obs, just sent the same: #54. Glad to see you've tested it, too. ;-)

**Comment by simison** [comment]
@sebastiannm Perhaps add some mentions on readme how placeholder works after this change?

**Comment by sebastiannm** [comment]
@simison It doesn't change anything about how it works, it just makes sure that placeholder is not set to false (which I do and would give me errors using this plugin)

**Comment by simison** [comment]
@sebastiannm :thumbsup:

File: package.json (hunk 1 above)
Comment (thijsw) [inline package.json]: I will take care of it; I like this kind of proactivity of contributors :+1:

**Comment by thijsw** [comment]
Thanks! Merged this contribution along with #54 and published a new release for it :)

**Pull request closed without merging**